src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,23 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private String model;
+    private String color;
+    private Engine engine;
+    private String driveUnit;
+    private int numberOfSeats;
+
+    public Car(String model, String color, Engine engine, String driveUnit, int numberOfSeats) {
+        this.model = model;
+        this.color = color;
+        this.engine = engine;
+        this.driveUnit = driveUnit;
+        this.numberOfSeats = numberOfSeats;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+        return new Car(model, color, engine, driveUnit, numberOfSeats);
```

Comment:
you will get a new Car but with the same engine

Reply:
ok

Reply:
that's not okey, that's problem. Imagine you have two cars but with one engine